lib/ansible/modules/windows/win_domain_group.ps1

```diff
@@ -0,0 +1,354 @@
+#!powershell
+# This file is part of Ansible
+#
+# (c) 2017, Jordan Borean <jborean93@gmail.com>, and others
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# WANT_JSON
+# POWERSHELL_COMMON
+
+$ErrorActionPreference = "Stop"
+
+$params = Parse-Args -arguments $args -supports_check_mode $true
+$check_mode = Get-AnsibleParam -obj $params -name "_ansible_check_mode" -type "bool" -default $false
+$diff_mode = Get-AnsibleParam -obj $Params -name "_ansible_diff" -type "bool" -default $false
+
+$name = Get-AnsibleParam -obj $params -name "name" -type "str" -failifempty $true
+$display_name = Get-AnsibleParam -obj $params -name "display_name" -type "str"
+$username = Get-AnsibleParam -obj $params -name "username" -type "str"
```

Comment:
That's a good point, usually *username* and *password* are prefixed in cases where more than one set is needed, e.g. *get_url* and *win_get_url* has a *url_username*/*url_password* and a *proxy_username*/*proxy_password*. Most modules that only require a single set, only use *username*/*password*.

If there's no possible other set imaginable I would definitely keep *username*/*password* as an alias for convenience. But personally I think *domain_admin_username* and *domain_admin_password* are not convenient. In the context of *win_domain_group* it seems logical this is an account with domain admin rights.